src/components/UseScript.vue

```diff
@@ -93,24 +107,51 @@ export default class UseScript extends Vue {
     this.loadData()
   }
 
+  private readonly defaultSchemaTypes = {
+    protobuf: {
+      extension: 'proto',
+    },
+    avro: {
+      extension: 'avsc',
+    },
+  }
+
   private async loadData() {
     const { scriptService } = useServices()
     const functions: ScriptModel[] | [] = (await scriptService.getAllFunction()) ?? []
     const schemas: ScriptModel[] | [] = (await scriptService.getAllSchema()) ?? []
     this.functions = functions
-    this.schemas = schemas
+
+    this.schemas =
+      this.currentSchemaType == null
```

Comment:
```suggestion
      this.currentSchemaType === null
```

Although `==` is fine, but `===` is a better lint for JS.